com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java

```diff
@@ -169,80 +170,91 @@ private void installOrUpdate(File outputFile) throws Exception {
 			isFragment = fragmentHost != null;
 
 			bsn = bundle.getBsn();
+
+			if(isFragment) {
+				hostBSN =
+						new Parameters(fragmentHost).keySet().iterator().next();
```

Comment:
hey fist , there should not be multiple parameters in Fragment-Host attribute , see https://github.com/bndtools/bnd/blob/master/org.osgi.impl.bundle.repoindex.lib/src/org/osgi/service/indexer/impl/BundleAnalyzer.java#L418
and there are no standard way to get the host bsn ,see https://github.com/eclipse/birt/blob/bab46c1570900bf70d4ab750c8be3cc344553882/build/org.eclipse.birt.build/src/org/eclipse/birt/build/framework/Bundle.java#L170-L181